logstash-codec-protobuf.gemspec

```diff
@@ -0,0 +1,26 @@
+Gem::Specification.new do |s|
+
+  s.name            = 'logstash-codec-protobuf'
+  s.version         = '0.1.2'
+  s.licenses        = ['Apache License (2.0)']
+  s.summary         = "This codec may be used to decode (via inputs) and encode (via outputs) protobuf messages"
+  s.description     = "This gem is a logstash plugin required to be installed on top of the Logstash core pipeline using $LS_HOME/bin/plugin install gemname. This gem is not a stand-alone program"
+  s.authors         = ["Inga Feick"]
+  s.email           = 'inga.feick@trivago.com'
+  s.require_paths   = ["lib"]
+
+  # Files
+  s.files = Dir['lib/**/*','spec/**/*','vendor/**/*','*.gemspec','*.md','CONTRIBUTORS','Gemfile','LICENSE','NOTICE.TXT']
+
+  # Tests
+  s.test_files = s.files.grep(%r{^(test|spec|features)/})
+
+  # Special flag to let us know this is actually a logstash plugin
+  s.metadata = { "logstash_plugin" => "true", "logstash_group" => "codec" }
+
+  # Gem dependencies
+  s.add_runtime_dependency 'logstash-core', '>= 1.4.0', '< 3.0.0'
```

Comment:
This would not work for 5.0. See dependency here: https://github.com/logstash-plugins/logstash-input-kafka/blob/master/logstash-input-kafka.gemspec#L28